<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_employes</class>
 <widget class="QMainWindow" name="gestion_employes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1364</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion_employes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1360</width>
      <height>740</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/qt/3.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/qt/Rectangle 10.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="button_employes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 85, 98);
</string>
    </property>
    <property name="text">
     <string>Employés</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_client">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 85, 98);
</string>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_produit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 85, 98);
</string>
    </property>
    <property name="text">
     <string>Produit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_fournisseur">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 85, 98);
</string>
    </property>
    <property name="text">
     <string>Fournisseur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reparation">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(0, 85, 98);
</string>
    </property>
    <property name="text">
     <string>Réparation</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>1095</width>
      <height>645</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="CIN_e">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Gestion des employés</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_e">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>100</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>80</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom_e">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>100</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>80</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="email_e">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>100</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="homme">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>190</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Homme</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="femme">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>160</y>
        <width>97</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Femme</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>RIB</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="RIB_e">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Rôle</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="role_e">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Salaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="salaire_e">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>56</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(184, 166, 73);</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_e">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(184, 166, 73); 
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(184, 166, 73);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mdp_e">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid rgb(184, 166, 73); 
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Date d'embauche</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_naissance">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>160</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="date_embauche">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>160</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boutton_ajouter">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_p1">
      <property name="geometry">
       <rect>
        <x>1060</x>
        <y>610</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);
</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boutton_modifier">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>210</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
border-color: rgb(0, 0, 0);
color: rgb(184, 166, 73);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border: 2px solid rgb(184, 166, 73); 
</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1111</width>
        <height>651</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/new/prefix1/qt/Rectangle 14.png</pixmap>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>991</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid rgb(0, 85, 98);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>550</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_e_supp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>570</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="boutton_supprimer">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>560</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche_nom">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>274</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #107D69;
width: 200px;
border-radius: 10px;</string>
      </property>
      <property name="placeholderText">
       <string>recherche par nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chercher_nom">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>274</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
border-color: rgb(0, 0, 0);
color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border: 2px solid rgb(0, 85, 98); 

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/new/prefix1/qt/chercher.png</normaloff>:/new/prefix1/qt/chercher.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>280</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 15px;
border-color: rgb(0, 0, 0);
color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border: 2px solid rgb(0, 85, 98); 

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/new/prefix1/qt/refresh.png</normaloff>:/new/prefix1/qt/refresh.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="button_trier">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>560</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
border-color: rgb(0, 0, 0);
color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border: 2px solid rgb(0, 85, 98);
</string>
      </property>
      <property name="text">
       <string>trier selon date d'embauche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_genererPDF">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>560</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
border-color: rgb(0, 0, 0);
color: rgb(0, 85, 98);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);
border: 2px solid rgb(0, 85, 98);
</string>
      </property>
      <property name="text">
       <string>Generer PDF</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>CIN_e</zorder>
     <zorder>label_14</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>nom_e</zorder>
     <zorder>label_6</zorder>
     <zorder>prenom_e</zorder>
     <zorder>label_7</zorder>
     <zorder>email_e</zorder>
     <zorder>label_8</zorder>
     <zorder>homme</zorder>
     <zorder>femme</zorder>
     <zorder>label_9</zorder>
     <zorder>RIB_e</zorder>
     <zorder>label_10</zorder>
     <zorder>role_e</zorder>
     <zorder>label_11</zorder>
     <zorder>salaire_e</zorder>
     <zorder>label_12</zorder>
     <zorder>id_e</zorder>
     <zorder>label_13</zorder>
     <zorder>mdp_e</zorder>
     <zorder>label_15</zorder>
     <zorder>label_16</zorder>
     <zorder>date_naissance</zorder>
     <zorder>date_embauche</zorder>
     <zorder>boutton_ajouter</zorder>
     <zorder>next_p1</zorder>
     <zorder>boutton_modifier</zorder>
     <zorder>tableView</zorder>
     <zorder>label_21</zorder>
     <zorder>id_e_supp</zorder>
     <zorder>boutton_supprimer</zorder>
     <zorder>recherche_nom</zorder>
     <zorder>chercher_nom</zorder>
     <zorder>refresh</zorder>
     <zorder>button_trier</zorder>
     <zorder>button_genererPDF</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>500</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/new/prefix1/qt/Rectangle 13.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Listes des employés</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_p2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>460</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);
</string>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pred_p0">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>460</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);
</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>500</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/new/prefix1/qt/Rectangle 13.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>501</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 85, 98);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pred_p1">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>460</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(184, 166, 73);
</string>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1364</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
